lib/config-migrator.js

```diff
@@ -0,0 +1,61 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default class ConfigMigrator {
```

Comment:
Export a function instead? `migrateConfiguration`, or maybe even just `migrate`?
Then we can avoid the static functions being exported etc.